docs/advanced/customization-guide.md

```diff
@@ -493,6 +493,7 @@ The following features are available for matching:
 | **`cpu.sst`**    | attribute    |          |            | Intel SST (Speed Select Technology) capabilities
 |                  |              | **`bf.enabled`** | bool | `true` if Intel SST-BF (Intel Speed Select Technology - Base frequency) has been enabled, otherwise does not exist
 | **`cpu.topology`** | attribute  |          |            | CPU topology related features
+| **`cpu.model`** | attribute  |          |            | CPU model related attributes
```

Comment:
nit: the "Feature" column should be in alphabetical order (I hope it is atm) so move this before `cpu.pstate`.

Also, could you list the available "elements" (i.e. family, vendor_id and id), similar how they are listed with other features (like `cpu.pstate`)